app/src/androidTest/java/com/example/roome/EditProfileWoutPictureTest.java

```diff
@@ -0,0 +1,410 @@
+package com.example.roome;
```

Comment:
I would add a documentation for each method so it would be easy for others to understand.
לא חובה - רק אם יש לך כוח